apps/addons/cron.py

```diff
@@ -453,9 +453,7 @@ def reindex_addons(index=None, addon_type=None):
     from . import tasks
     ids = (Addon.objects.values_list('id', flat=True)
            .filter(_current_version__isnull=False,
-                   status__in=amo.VALID_STATUSES,
-                   is_listed=True,
-                   disabled_by_user=False))
```

Comment:
This was there before the commit you're talking about in the bug (https://github.com/mozilla/olympia/commit/ae2e2cb). I guess it was also faulty?